src/types.ts

```diff
@@ -6,3 +6,34 @@ export type Character = {
   lastLocation: string;
   gender: string;
 };
+
+export type CharacterFromAPI = {
+  id: number;
+  name: string;
+  status: string;
+  species: string;
+  type: string;
+  gender: string;
+  image: string;
+  origin: {
+    name: string;
+    url: string;
+  };
+  location: {
+    name: string;
+    url: string;
+  };
+  episode: string[];
+  url: string;
+  created: string;
+};
+
+export type AllCharactersFromAPI = {
+  results: CharacterFromAPI[];
```

Comment:
The API has a different order, but I guess it doesn't matter too much. However, I'd prefer this key to be the last in this type